topics/08_Known-Issues/01_Known_Issues_and_Limitations_2016_Volume_2.md

```diff
@@ -391,6 +391,22 @@ When you use the minValue and maxValue options in the igEditors, you should be a
 ![](../images/images/negative.png)
 			</td>
         </tr>
+		
+		<tr>
+            <td>
+Spinning up when the maximum value is reached and spinWrapAround option is true is not possible if  maxValue is not set with dataMode being float, long or double
+			</td>
+
+            <td>
+When you use the [`minValue`](%%jQueryApiUrl%%/ui.ignumericeditor#options:minValue)option in the igEditors without setting the [`maxValue`](%%jQueryApiUrl%%/ui.ignumericeditor#options:maxValue) option, you should keep in mind that the [`maxValue`](%%jQueryApiUrl%%/ui.ignumericeditor#options:maxValue) for the editor will be set to the maximum number for the [`dataMode`](%%jQueryApiUrl%%/ui.ignumericeditor#options:dataMode) that is selected. When you also have spinWrapAround set to true and you try to spin up u when the maximum value is already reached, trying to set the value to the opposite side of the limit, the value will not change. This is in case the  [`dataMode`](%%jQueryApiUrl%%/ui.ignumericeditor#options:dataMode) option is set to any of the following values: float, long or double. 
+The reason for this behavior is that the maximum values for those data modes are big numbers that are represented in scientific notation in JavaScript. 
+To bypass this limitation, you can set the [`maxValue`](%%jQueryApiUrl%%/ui.ignumericeditor#options:maxValue) to a number that is not represented in scientific notation by JavaScript or you can set the  [`scientificFormat`](%%jQueryApiUrl%%/ui.ignumericeditor#options:scientificFormat) option in the igEditor.  
```

Comment:
- Make it set both min and max to guarantee the spin range.
- Same as above, swap "igEditor" for just "editor" or Numeric, Percent and Currency editors.